addons/point_of_sale/static/src/js/chrome.js

```diff
@@ -745,6 +745,36 @@ var Chrome = PosBaseWidget.extend(AbstractAction.prototype, {
 
         popup.appendTo(this.$el);
     },
+    json_error: function(err){
+        var self = this;
+        var body = "There are conflicts between the localStorage and the loaded data. " +
+            "Possibly there where changes made in the database after first initialisation of this PoS session.\n\n" +
```

Comment:
Avoid using terms like "localStorage" in error messages, only developers understand that.
Don't forget to translate the error messages.
"Changes were made in the..." instead of "Possible there where..."